gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -202,13 +201,13 @@ private static void configureErrorProneOptions(
         }
 
         errorProneOptions.getDisableWarningsInGeneratedCode().set(true);
-        String projectPath = project.getProjectDir().getPath();
-        String separator = Pattern.quote(Paths.get(projectPath).getFileSystem().getSeparator());
+        // don't want backslashes on windows to break our regex
+        String separator = File.separatorChar == '\\' ? Pattern.quote("\\") : File.separator;
         errorProneOptions
                 .getExcludedPaths()
                 .set(String.format(
-                        "%s%s(build|src%sgenerated.*)%s.*",
-                        Pattern.quote(projectPath), separator, separator, separator));
+                        ".*(build%sgenerated%ssources|src%sgenerated.*)%s.*",
```

Comment:
This does not seem to handle cases that previously were excluded such as `build/metricSchema/generated_src` used by metric-schema which is causing the baseline bump excavator on tritium to fail compilation: https://app.circleci.com/pipelines/github/palantir/tritium/906/workflows/7c65b97b-b475-46fc-a1e1-b5a4bf7198a0/jobs/11565

Filed https://github.com/palantir/gradle-baseline/issues/1570